core_perception/lidar_localizer/launch/ndt_matching.launch

```diff
@@ -28,8 +28,8 @@
     <param name="get_height" value="$(arg get_height)" />
     <param name="use_local_transform" value="$(arg use_local_transform)" />
     <param name="output_log_data" value="$(arg output_log_data)" />
+    <param name="output_tf_frame_id" value="$(arg output_tf_frame_id)" />
     <param name="gnss_reinit_fitness" value="$(arg gnss_reinit_fitness)" />
-    <param name="base_frame" value="$(arg base_frame)" />
```

Comment:
Take a look at carma-platform/carma/launch/localization.launch for this launch file being called and make sure the argument is switched from base_frame to output_tf_frame_id if it was provided previously.